test/egg.test.js

```diff
@@ -415,6 +415,10 @@ describe('test/egg.test.js', () => {
       const wrapped = app.toAsyncFunction(fn);
       return wrapped(true).then(res => assert(res === true));
     });
+
+    it('not translate an obj that is not a function', () => {
+      assert.throws(() => app.toAsyncFunction(123, true), null, '\'fn\' should be a function.');
```

Comment:
https://nodejs.org/api/assert.html#assert_assert_throws_block_error_message

Is this should be `assert.throws(() => app.toAsyncFunction(123, true), /^'fn' should be a function\.$/)`

Reply:
OK. Fixed it :)